docs/devel/luns.md

```diff
@@ -0,0 +1,56 @@
+Direct LUNs migration
+==============
+
+Summary
+-------
+
+VM contains disks. Some of them might be connected as a direct LUN.
+When a user wish to re-use the LUN disks, he needs them attached to the VM.
+The LUN should be used in a single place, or else, it will cause a data corruption.
+The storage where the LUNs reside, needs to be reachable to the VM environment.
+This feature page mostly about import from oVirt.
```

Comment:
as far as I know we never supported importing LUNs from vsphere (definitely not when importing through RHV, most probably also not from IMS) so let's scope this only to oVirt/RHV